src/iguana/algorithms/clas12/LeptonIDFilter/Algorithm.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include "iguana/algorithms/Algorithm.h"
+
+namespace iguana::clas12 {
+
+  /// Set of lepton ID variables
+  struct LeptonIDVars {
+      double P;
+      double Theta;
+      double Phi;
+      double SFpcal;
+      double SFecin;
+      double SFecout;
+      double m2pcal;
+      double m2ecin;
+      double m2ecout;
+      double score;
```

Comment:
Either each one of these members needs to be documented, or you can use `@doxygen_off` and `@doxygen_on` to avoid having to write documentation here.